research/conditional/utils/argparse.py

```diff
@@ -144,7 +156,7 @@ def introduce_parser_arguments(
     parser.add_argument("--n_gpus", type=int, default=1)
 
     # Logging parameters
-    parser.add_argument("--logger_types", type=str, required=True)
+    parser.add_argument("--logger_types", type=str, default=None)
```

Comment:
Why?

Reply:
checkoiint manager chandle loger automatially when enable and works only with neptune. Have added ags checks.

Reply:
IMO It would be better to ask user to specify neptune if he want to use checkpoint manager, If I specify None I do not want to implicite use neptune.

Reply:
done